twitter_clone/main.py

```diff
@@ -25,5 +54,231 @@ def decorated_function(*args, **kwargs):
         return f(*args, **kwargs)
     return decorated_function
 
-
 # implement your views here
+@app.route('/')
+def base():
+    '''
+    Redirect user to login page.
+    '''
+    # If not logged in, redirect to login page
+    if 'logged_in' not in session:
```

Comment:
You could also have used the `login_required decorator` on this function, but this works just as well